portal-web/test/functional/com/liferay/portalweb/tests/enduser/analyticscloud/DeleteSegments.testcase

```diff
@@ -76,10 +90,16 @@ definition {
 
 		task ("Delete the criteria") {
 			ACSegments.deleteSegmentCriteria(criteriaName = "birthDate");
+
+			ACSegments.saveSegment();
 		}
 
-		task ("View the deleted criteria") {
-			ACSegments.viewDeletedCriteria();
+		task ("View the segment criteria") {
+			ACSegments.viewSegmentCriteriaIndividualAttributes(attributeName = "createDate");
+
+			AssertTextNotEquals(
+				locator1 = "ACSegments#SEGMENT_CRITERIA_CARD",
+				value1 = "birthDate");
```

Comment:
I think this path already exists

`ACSegments.assertCriteriaTable(criteriaName = "birthDate");`